### DISCUSSION TITLE
Is it OK to port Squants to another programming language?

### DISCUSSION THREAD

**Discussion by varnenec** [typelevel/squants#449] [General]
Hi,

I was considering porting Squants to Kotlin, and calling the result `kosquants` or `ksquants`. I wanted to make sure I do right by the Squants community, so I have two questions:

* Are you OK with a project to port Squants to another language in general?
* Are you OK if the name of the port contains `squants` (mostly to help users understand the relationship), or would you rather that the name is unrelated, and the original Squants project is acknowledged as inspiration in some other way?

Thanks!

**Comment by cquiroz** [comment]
I don't sae a reason why not. the license is liberal enough, maybe you could link back to `squants` giving credit
